### PR TITLE
Bump the ubuntu k8s version to 0.19.3

### PR DESCRIPTION
**Pull request by WIZARD-CXY**
Changes:
1 update the etcd version to 2.0.12 and k8s version to 0.19.3
2 starting using service account token for ubuntu

Tested OK.

### CODE REVIEW

**Comment by WIZARD-CXY** [comment]
Now I'm working on making the k8s more secure to use by using https on ubuntu , but I am not very good at security stuff. I think I will make it work before 1.0!

**Comment by zmerlynn** [comment]
LGTM

**Comment by WIZARD-CXY** [comment]
@zmerlynn Waiting for a merge

**Comment by zmerlynn** [comment]
I need leads to LGTM to merge.

**Comment by gmarek** [comment]
Impacts users: #10480

File: cluster/ubuntu/build.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ cp flannel-${FLANNEL_VERSION}/flanneld binaries/minion
 
 # ectd
 echo "Download etcd release ..."
-ETCD_VERSION="v2.0.9"
+ETCD_VERSION="v2.0.12"
```
Comment (thockin) [inline cluster/ubuntu/build.sh]: Note that we have not qualified 2.0.12 ourselves.  I don't know if we want to STOP you from doing this, but you have to know that any problems you hit, we can't help yet.

@lavalamp @dchen1107 how aggressively do we want to discourage this?

Comment (zmerlynn) [inline cluster/ubuntu/build.sh]: We've been running 2.0.12 for a week now: https://github.com/GoogleCloudPlatform/kubernetes/pull/10140

Comment (thockin) [inline cluster/ubuntu/build.sh]: Oh, I apologize, I thought we backed that out.

**Comment by thockin** [comment]
LGTM  @erictune

**Comment by WIZARD-CXY** [comment]
In my lab etcd 2.0.12 is working well without a problem for a week now

**Comment by WIZARD-CXY** [comment]
Just update the kube-dns to v1 api and tested ok! @gmarek 
@thockin I use the kube2sky `kube_master_url` flag to help it connect to apiserver insecurely to get it work now.

**Comment by WIZARD-CXY** [comment]
@erictune wait for a merge~~~

**Comment by gmarek** [comment]
There's a holiday in US (well - extended holiday in Google), so I'm afraid that @erictune might not be available until Monday...

**Comment by WIZARD-CXY** [comment]
😅😅 I want a holiday

May the force be with you

**Comment by bprashanth** [comment]
Looks like this just fixes the ubuntu scripts, in which case it would be nice to have for 1.0. Waiting on Eric's review and ok-to-merge.

**Comment by WIZARD-CXY** [comment]
Roger that

**Comment by WIZARD-CXY** [comment]
@erictune wait for your review .Thx

**Comment by WIZARD-CXY** [comment]
@thockin @zmerlynn@gmarek Maybe @erictune is busy.Please change another assignee and do a merge.Another pr add dns to docker based k8s . https://github.com/GoogleCloudPlatform/kubernetes/pull/10829 is depending on it

**Comment by WIZARD-CXY** [comment]
Just wait for too long....Please

**Comment by zmerlynn** [comment]
This needs a rebase.

**Comment by WIZARD-CXY** [comment]
roger that

**Comment by WIZARD-CXY** [comment]
@zmerlynn just rebased and do a forced push.Tested OK!

**Comment by zmerlynn** [comment]
@thockin: Can we get an ok-to-merge? This is way stale, community, and won't hit core

**Comment by WIZARD-CXY** [comment]
I noticed kube-dns had just upgraded to v6. I am currently using v4. It works just fine.

**Comment by WIZARD-CXY** [comment]
@thockin can you give a ok-to-merge?

File: cluster/ubuntu/util.sh [hunk 2]
```diff
@@ -181,6 +181,16 @@ function verify-minion(){
   done
 }
 
+function genServiceAccountsKey() {
```
Comment (erictune) [inline cluster/ubuntu/util.sh]: What is going on here?

Comment (erictune) [inline cluster/ubuntu/util.sh]: Oh, right. looks good.

**Comment by erictune** [comment]
LGTM.  Thanks for updating!

**Comment by gmarek** [comment]
Needs rebase.

**Comment by WIZARD-CXY** [comment]
@gmarek @erictune rebased. good to go!

**Comment by gmarek** [comment]
We have two lead LGTMs, I'm merging it despite lack of ok-to-merge.

**Comment by WIZARD-CXY** [comment]
thx

**Pull request merged**